perf-dashboard/clientlibs-download/python_clientlibs_download.py

```diff
@@ -45,6 +45,7 @@
         'google-cloud-translate',
         'google-cloud-dns',
         'google-cloud-videointelligence',
+        'pandas-gbq',
```

Comment:
Since this isn't a Veneer library, can you add another subkey and add pandas-gbq to that?  Maybe `THIRD_PARTY_TABLE_NAME`.